src/index.ts

```diff
@@ -57,15 +60,19 @@ export const generateUri = (
  * @param timestamp optional, timestamp used for deterministic unit tests (defaults to current timestamp)
  * @returns token
  */
-export const generateToken = (secret: string, timestamp = Date.now()) => {
+export const generateToken = (
+  secret: string,
+  timestamp = Date.now(),
+  algorithm: hashAlgorithm = 'SHA1'
```

Comment:
Given `timestamp` is only used for unit tests, should we move `algorithm` above `timestamp` to avoid requiring `timestamp` parameter?

Reply:
This could potentially break callers that are utilizing `timestamp`.  I'm actually using it in product code for a bit of an edge case.  But happy to re-order if that's your preference.  I do recognize the vast majority of callers won't utilize this parameter.

Reply:
Updated